contracts/src/v0.8/tests/OCR2DRClientTestHelper.sol

```diff
@@ -10,9 +10,7 @@ contract OCR2DRClientTestHelper is OCR2DRClient {
 
   event FulfillRequestInvoked(bytes32 requestId, bytes response, bytes err);
 
-  constructor(address _oracle) {
-    setOracle(_oracle);
-  }
+  constructor(address _oracle) OCR2DRClient(_oracle) {}
```

Comment:
I think that we should be consistent with naming function parameters.  Some function params are prefixed with `_` and some are not.